### DISCUSSION TITLE
Ignore host but see its activity

### DISCUSSION THREAD

**Discussion by borisspektor** [mitmproxy/mitmproxy#7188] [Ideas]
I am using mitmweb. Currently when ignoring hosts the requests made to that host are completely ignored. It would be super useful to see that a request was made but was ignored by the proxy.
Is there a way to achieve this behavior?

**Comment by Prinzhorn** [comment]
Sounds like https://github.com/mitmproxy/mitmproxy/pull/6720 ?

**Reply by borisspektor** [reply]
Yes! Thank you. I will close this discussion.
